### ISSUE TITLE
accessibilityLabel is not working in XCUITest Cases (test cases) (accessibilityIdentifier)

### ISSUE THREAD

**Issue by Mushrankhan** [hackiftekhar/IQKeyboardManager#1822]
**Describe the bug**
problem in XCUITestCases.
seems like accessibilityLabel for rightConfig is not working, specially I think when multilingual. 
you can check the screenshots I have attached.
1) debugged, it is assigning "Done".
2) runtime it is not identified by accessibilityLabel, I think given text value is label value, accessibilityLabel missing.
Resolution :
I think you can add "accessibilityIdentifier" in config and assign, so components can be identified by "accessibilityIdentifier" at runtime in xcuitest cases.

thanks a lot.

screenshots :
![1](https://user-images.githubusercontent.com/16305446/124374361-4f739480-dcb8-11eb-8e64-bf481d0f4fa2.png)
![2](https://user-images.githubusercontent.com/16305446/124374366-54384880-dcb8-11eb-8346-623d512c51f0.png)

**Comment by Mushrankhan** [comment]
thanks for resolving.. keep up the good work 👍
